src/diagrams/flowchart/parser/flow.jison

```diff
@@ -380,6 +381,8 @@ vertex:  idString SQS text SQE
         {$$ = $1;yy.addVertex($1,$3,'stadium');}
     | idString SUBROUTINESTART text SUBROUTINEEND
         {$$ = $1;yy.addVertex($1,$3,'subroutine');}
+    | idString VERTEX_WITH_PROPS_START ALPHA COLON ALPHA PIPE text SQE
```

Comment:
I'm not sure if there is a better token (or rule) than `ALPHA` for the identifier and/or value. It does not allow to add a property `thick-borders`, but `thickBorders` would work. That's fine to me. Further, the rule could be changed in the future if required.